contracts/truefi2/LoanFactory2.sol

```diff
@@ -53,12 +66,24 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
         lender = _lender;
         admin = msg.sender;
         liquidator = _liquidator;
+
+        adjustmentCoefficient = 25;
     }
 
     function setAdmin() external {
         admin = 0x16cEa306506c387713C70b9C1205fd5aC997E78E;
     }
 
+    function setAdjustmentCoefficient(uint256 newCoefficient) external {
+        require(msg.sender == admin, "LoanFactory: Only admin can set adjustment coefficient");
+        adjustmentCoefficient = newCoefficient;
+        emit AdjustmentCoefficientChanged(newCoefficient);
+    }
+
+    function fixTermLoanAdjustment(uint256 term) public view returns (uint256) {
+        return term.div(30 days).mul(adjustmentCoefficient);
```

Comment:
```suggestion
        return term.mul(adjustmentCoefficient).div(30 days);
```

Reply:
This is intentional, as it allows to change adjustment non-continuously